_docs/Threat-Model.md

```diff
@@ -27,38 +27,38 @@ We distinguish between four attacker types:
 3. Qabel servers,
 4. Outside attackers:
   1. Outside attacker who can eavesdrop traffic of the client,
-  2. Outside attacker who can eavesdrop traffic of a Qabel drop server,
-  3. Outside attacker who can eavesdrop traffic of a Qabel storage server, 
+  2. Outside attacker who can eavesdrop traffic of a Qabel Drop server,
+  3. Outside attacker who can eavesdrop traffic of a Qabel Block server, 
   4. Outside attacker who can eavesdrop traffic of the public Internet (e.g., *DE-CIX Frankfurt*).
 
 ![Attacker Types](/images/attackerTypes.png)
 
 ## Capabilities of Attackers
 ### 0. Everyone
-The stored encrypted data is publicly available. Thus everybody knows the number and sizes of all files and the estimated number of all folders of every prefix.
+Writing and reading of drop messages is not restricted. Currently downloading from the Qabel Block server is restricted to authenticated accounts.
 
 ### 1. Contact
 A user *B* learns one identity (drop ID, public key and alias) of a user *A* during the contact. When receiving a share user *B* learns the directory structure of the shared directory and its files or the shared file. This implies that *B* learns the name of *A's* prefix the share is stored in.
 
 ### 2. Qabel User 
-An attacker has no advanced capabilities regarding security by being a Qabel user.
+The stored encrypted data is accessible for all Qabel accounts. Thus every user knows the number and sizes of all files and the estimated number of all folders of every prefix.
```

Comment:
There is no way for attackers to list all files, he needs to guess the file names which are long and random. Only the server knows the file names.